scripts/linters/general_purpose_linter.py

```diff
@@ -232,6 +232,17 @@
             'typings/guppy-defs-b5055b963fdbea5c6c1e92dbf58fdaf3ea0cd8ba.d.ts',
             'core/templates/services/UpgradedServices.ts'),
         'excluded_dirs': ()
+    },
+    {
+        'regexp': re.compile(
+            r'eslint-(disable|enable)(-next-line)?'
+            ' @typescript-eslint/no-explicit-any'),
+        'message': (
+            'Please do not define any types. You can refer '
```

Comment:
any --> "any"

Great idea to refer to the wiki page -- thanks!

Reply:
done